data/policies/derive.cas

```diff
@@ -200,3 +200,21 @@ virtual resource alias_parent2 inherits alias_parent1 {
 
 @alias(alias2)
 resource my_res inherits alias_parent2 {}
+
+virtual resource g {
+	fn func(domain source) {
+		allow(source, this, file, read);
+	}
+}
+
+virtual resource h {
+	@alias(func)
+	fn func2(domain source) {
+		allow(source, this, file, write);
+	}
+}
+
+@alias(j)
+virtual resource i inherits g, h {}
```

Comment:
In the case of test code, I'm not sure it's any worse than foo/bar/baz.  I used it here because it's more scalable without looking up the extensions to the foo/bar/baz system.